### PR TITLE
fix(CORE-128) basic notifications load leveling

### PR DESCRIPTION
**Pull request by tsmbl**
See jira for details: https://dialect.atlassian.net/browse/CORE-128?focusedCommentId=10012

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { Logger } from 'nestjs-pino';
 import { PrismaExceptionInterceptor } from './middleware/prisma-error-interceptor';
 import admin from 'firebase-admin';
 
-bootstrap();
+void bootstrap();
```
Comment (fsher) [inline src/main.ts:15]: Never used `void` this way. Could you please clarify why are you doing this?

**Review by fsher — APPROVED** [review]
LGTM, but no expert here.

File: src/main.ts (hunk 1 above)
Comment (tsmbl) [reply src/main.ts:16]: This fix is proposed by eslint

Comment (tsmbl) [reply src/main.ts:16]: Added comment

**Review by zhelezkov — APPROVED** [review]
lgtm gtg

**Pull request merged**